go.mod

```diff
@@ -32,6 +32,7 @@ replace (
 
 // pinned to grpc v1.26.0
 replace (
+	github.com/golang/latestprotobuf => github.com/golang/protobuf v1.4.3
```

Comment:
Let me know if there is a non ugly version of doing this.

Reply:
Not exactly sure why you did this. I see you've pushed protobuf below to `1.4.1`. Why not straight to `1.4.3` and that's that?

Reply:
we still have `github.com/golang/protobuf => github.com/golang/protobuf v1.3.2` so I don't know if this is done on purpose to stay on the old version. If I can bump that, that would be perfect.

Reply:
I'd suggest to `git blame` that line and ask the person. It should likely be fine.